compute/compute-node.Dockerfile

```diff
@@ -139,7 +144,7 @@ RUN case $DEBIAN_VERSION in \
 #########################################################################################
 FROM build-deps AS pg-build
 ARG PG_VERSION
-COPY vendor/postgres-${PG_VERSION} postgres
+COPY vendor/postgres-${PG_VERSION:?} postgres
```

Comment:
What's this for?

Reply:
That is for the cases when `PG_VERSION` is unset (maybe mistype, or may be some issue in automation).
Right now in that case build process stops with error message that it fails to copy some files.
After adding `:?` to vairable, it will fail with error message like:
```
3.684 /bin/bash: line 1: PG_VERSION: parameter null or not set
```
which gives more context of the issue.

Unfortunately full syntax is not supported (when I tried), so putting error message there doesn't work
```
❯ echo COPY vendor/postgres-${PG_VERSION:?ARG PG_VERSION must be set to the one of the PG Versions: v14, v15, v16, v17}
zsh: PG_VERSION: ARG PG_VERSION must be set to the one of the PG Versions: v14, v15, v16, v17
```

Reply:
Oh, nice! I always forget to set PG_VERSION at first :-).